bcda/responseutils/writer.go

```diff
@@ -22,4 +23,105 @@ func WriteError(outcome *fhirmodels.OperationOutcome, w http.ResponseWriter, cod
 	}
 }
 
-func WriteCapabilityStatement() {}
+func CreateCapabilityStatement(reldate time.Time, relversion string) *fhirmodels.CapabilityStatement {
+	usecoors := true
+	baseurl := "<host>"
+	statement := &fhirmodels.CapabilityStatement{
+		Status:       "active",
+		Date:         &fhirmodels.FHIRDateTime{Time: reldate, Precision: fhirmodels.Date},
+		Publisher:    "Centers for Medicare & Medicaid Services",
+		Kind:         "capability",
+		Instantiates: []string{"https://api.bluebutton.cms.gov/v1/fhir/metadata"},
```

Comment:
This url is for the BB 2.0 frontend, but we are interacting with the BB 2.0 backend. Can you either drop the `Instantiates` key or have it use the value of the env var [`BB_SERVER_LOCATION`](https://github.com/CMSgov/bcda-app/blob/master/Dockerfiles/Dockerfile.bcda#L19)?

Reply:
cool, will do